packages/lesswrong/components/ultraFeed/FeedContentBody.tsx

```diff
@@ -231,9 +261,49 @@ const FeedContentBody = ({
     }
   };
 
-  const showContinueReadingLink = !applyLineClamp && isMaxLevel && wasTruncated && linkToDocumentOnFinalExpand;
+  const showContinueReadingAction = !applyLineClamp && isMaxLevel && wasTruncated;
   const isClickableForExpansion = !isMaxLevel;
 
+  const continueReadingAction = (() => {
```

Comment:
would strongly prefer this to be explicitly defined as a function outside the component